codegen.mts

```diff
@@ -1,10 +1,143 @@
+import { defineConfig } from '@eddeee888/gcg-typescript-resolver-files';
 import { type CodegenConfig } from '@graphql-codegen/cli';
 import { addTypenameSelectionDocumentTransform } from '@graphql-codegen/client-preset';
 
 const config: CodegenConfig = {
   schema: './packages/services/api/src/modules/*/module.graphql.ts',
   emitLegacyCommonJSImports: true,
   generates: {
+    // Server preset
+    './packages/services/api/src': defineConfig({
+      add: {
+        './__generated__/types.next.ts': {
+          content: "import type { StripeTypes } from '@hive/stripe-billing';",
+        },
+      },
+      typeDefsFilePath: false,
+      resolverMainFileMode: 'modules',
+      resolverTypesPath: './__generated__/types.next.ts',
+      whitelistedModules: ['usage-estimation', 'admin'],
+      scalarsOverrides: {
+        DateTime: { type: 'string' },
+        Date: { type: 'string' },
+        SafeInt: { type: 'number' },
+        ID: { type: 'string' },
+      },
+      typesPluginsConfig: {
```

Comment:
This is kept the same as modules preset to keep the two generated versions the same